apps/clabs/store.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
if we need a Model for some persistency, please create the proper models instead of this new definition of store.